overlord/snapstate/snapmgr.go

```diff
@@ -102,6 +102,11 @@ func (ss *SnapSetup) TryMode() bool {
 	return ss.Flags&TryMode != 0
 }
 
+// RevertOp returns true if the operation is a revert
+func (ss *SnapSetup) RevertOp() bool {
```

Comment:
This got re-added because this branch needs to know if an operation was a refresh to a local version (which needs a data copy) or a revert to a local version (which does not copy data). Both the refresh and the revert code-path call doInstall() eventually and if there is a revision attached to the SnapSetup we need to know in addition to the revert if we need a data copy or not. Happy to do that in a different way of course, I could generate the tasks in `revertToRevision()` for example instead of calling doInstall() there, this would avoid this `RevertOp`. More ideas welcome of course!